packages/frontend-2/components/settings/workspaces/Billing.vue

```diff
@@ -1,75 +1,166 @@
-            to="mailto:hello@speckle.systems"
-            color="primary"
-          >
-            Talk to us
-          </FormButton>
-        </div>
-      </div>
+          <CommonCard v-if="isActivePlan" class="bg-foundation">
+            <div class="flex flex-row gap-x-4 items-center">
+              <p class="text-body-xs text-foreground-2 flex-1">
+                View invoices, edit payment details, and manage your subscription from
+                the billing portal
+              </p>
+              <FormButton
+                color="outline"
+                :icon-right="ArrowTopRightOnSquareIcon"
+                @click="openCustomerPortal"
+              >
+                Open billing portal
+              </FormButton>
+            </div>
+          </CommonCard>
 
-      <div v-if="isBillingIntegrationEnabled" class="flex flex-col space-y-10">
-        <SettingsSectionHeader title="Start payment" class="pt-10" subheading />
-        <div class="flex items-center">
-          <div class="flex-1 flex-col pr-6 gap-y-1">
-            <p class="text-body-xs font-medium text-foreground">Billing cycle</p>
-            <p class="text-body-xs text-foreground-2 leading-5 max-w-md">
-              Choose an annual billing cycle for 20% off
-            </p>
+          <SettingsSectionHeader title="Price plans" subheading class="pt-4" />
+          <div class="flex items-center gap-x-4">
+            <div class="flex-col pr-6 gap-y-1">
+              <p class="text-body-xs font-medium text-foreground">Annual billing</p>
+              <p class="text-body-xs text-foreground-2 leading-5 max-w-md">
+                Choose annual billing for a 20% discount
+              </p>
+            </div>
+            <FormSwitch
+              v-model="isYearlyPlan"
+              :show-label="false"
+              name="annual billing"
+            />
+          </div>
+          <div class="grid grid-cols-1 md:grid-cols-3 gap-3">
+            <CommonCard
+              v-for="pricingPlan in pricingPlans"
+              :key="pricingPlan.name"
+              class="gap-y-4"
+            >
+              <h4 class="text-heading text-foreground capitalize">
+                {{ pricingPlan.name }}
+              </h4>
+              <FormButton
+                color="outline"
+                full-width
+                @click="onUpgradePlanClick(pricingPlan.name)"
+              >
+                Upgrade
+              </FormButton>
+            </CommonCard>
           </div>
-          <FormSwitch v-model="isYearlyPlan" :show-label="true" name="annual billing" />
         </div>
-        <div class="text-lg">Add the pricing table here</div>
-        <div class="flex justify-between">
-          <FormButton @click="teamCheckout">Team plan</FormButton>
-          <FormButton @click="proCheckout">Pro plan</FormButton>
-          <FormButton @click="businessCheckout">Business plan</FormButton>
+      </template>
+
+      <template v-else>
```

Comment:
This will be irrelevant when billing is enabled. @gjedlicska @benjaminvo is there even any benefit of keeping this in now?

Reply:
no, i think we should delete this